pkg/handler/handler_pod_test.go

```diff
@@ -19,14 +19,15 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/aws/amazon-eks-pod-identity-webhook/pkg/cache"
-	"github.com/aws/amazon-eks-pod-identity-webhook/pkg/containercredentials"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"testing"
 
+	"github.com/aws/amazon-eks-pod-identity-webhook/pkg/cache"
+	"github.com/aws/amazon-eks-pod-identity-webhook/pkg/containercredentials"
+
 	corev1 "k8s.io/api/core/v1"
 	"sigs.k8s.io/yaml"
```

Comment:
The changes in this file are just minor refactoring for better readability.

It's easier to view with whitespace diffs disabled:
https://github.com/aws/amazon-eks-pod-identity-webhook/pull/239/files?w=1